prow/pjutil/abort.go

```diff
@@ -36,35 +37,79 @@ type prowClient interface {
 // message.
 type ProwJobResourcesCleanup func(pj prowapi.ProwJob) error
 
+func refsForRefs(ref prowapi.Refs) refs {
+	var pulls []int
+	for _, pull := range ref.Pulls {
+		pulls = append(pulls, pull.Number)
+	}
+	return refs{
+		org:   ref.Org,
+		repo:  ref.Repo,
+		base:  ref.BaseRef,
+		pulls: pulls,
+	}
+}
+
 // jobIndentifier keeps the information required to uniquely identify a prow job
 type jobIndentifier struct {
-	job          string
-	organization string
-	repository   string
-	pullRequest  int
+	job  string
+	refs []refs
+}
+
+type refs struct {
```

Comment:
Why not reuse the `prowapi.Refs` type and just ignore the unneeded fields instead of creating a struct that has a subset of the fields?

Reply:
I originally wanted to use `refs` as the map key but couldn't (due to it having slices) -- this sounds good